packages/internal-lib-build/README.md

```diff
@@ -0,0 +1,13 @@
+# PWA Kit Internal tools 🛠️
```

Comment:
```suggestion
# PWA Kit Internal Tools
```

Use title case for h1s. For consistency with other README files in our repo, use emoji only on h2s (and put the emoji before the heading text, not after).